### ISSUE TITLE
[AXE-CORE]: LI elements must be contained in an UL or OL

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#21784]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix issue.
1. At message the accessibility specialist when issue is resolved.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement
Accessibility specialists discovered Nightwatch axe checks failing silently during a recent staging review. Two axe violations were discovered where list items were not contained in unordered or ordered lists. Links to pages below.

## Acceptance Criteria
- [ ] All instances of the error are fixed
- [ ] Axe-browser plugin or axe-core CLI scans show 0 Section 508, WCAG2A or WCAG2AA violations

## Environment
* https://www.va.gov/health-care/refill-track-prescriptions/
* https://www.va.gov/health-care/view-test-and-lab-results/

## Steps to Recreate
1. Open one of the listed URLs
1. Open axe browser plugin (assumes already installed in Chrome or Firefox)
1. Confirm the `<li> elements must be contained in a <ul> or <ol>` is present one or more times

**Comment by kelsonic** [comment]
PR is up: https://github.com/department-of-veterans-affairs/vets-website/pull/16602

**Comment by brianalloyd** [comment]
@1Copenut this fix should go out with today's deploy, can you validate fix is in place in support of ticket closure?  Thanks.

**Comment by 1Copenut** [comment]
@brianalloyd Yes, I'll give this a look tomorrow morning first thing and close the ticket when verified. I have high confidence Kelson's captured the ask.

**Comment by kelsonic** [comment]
Hey folks, sorry for the confusion, PR was just merged so it will go out with today's deployment 🙂 Moved the ticket back to validate

**Comment by brianalloyd** [comment]
Thanks @kelsonic appreciate the follow up.  will follow up post deployment to validate.  Thanks

**Comment by 1Copenut** [comment]
Fixes look good @kelsonic. Moving to close.

/cc @brianalloyd
